README.md

```diff
@@ -61,23 +61,33 @@ ReactJS based Presentation Library
 
 ## Getting Started
 
-The new best way to get started is by running `create-react-app my-presentation --scripts-version spectacle-scripts`. This will use `create-react-app` to create almost everything you need. This however, doesn't include publish scripts, and ejecting is required for fancy stuff.
+The best way to get started is by running
+
+npm
+
+`npm create-react-app my-presentation --scripts-version spectacle-scripts`
+
+yarn
+
+`yarn create react-app my-presentation --scripts-version spectacle-scripts`
+
+This will use `create react-app` to create almost everything you need. This however, doesn't include publish scripts, and ejecting is required for fancy stuff.
 
 The second best way to get started is by using the [Spectacle Boilerplate](https://github.com/FormidableLabs/spectacle-boilerplate).
 
+From here you can you go to [Development](#development) to get started.
+
 Alternatively, you can `npm install spectacle` and write your own build configurations. We also provide full UMD builds (with a `Spectacle` global variable) of the library at `dist/spectacle.js` and `dist/spectacle.min.js` for more general use cases. You could, for example, include the library via a script tag with: `https://unpkg.com/spectacle@VERSION/dist/spectacle.min.js`.
 
 Note that we have webpack externals for `react`, `react-dom`, and `prop-types`, so you will need to provide them in your upstream build or something like linking in via `script` tags in your HTML page for all three libraries. This comports with our project dependencies which place these three libraries in `peerDependencies`.
 
-But really, it is SO much easier to just use the boilerplate. Trust me.
-
 <a name="one-page"></a>
 
 ## One Page
 
-To aid with speedy development / kicking the tires on spectacle, we support using a simple boilerplate HTML page with a bespoke script tag that contains your entire presentation. The rest of the setup will take care of transpiling your React/ESnext code, providing Spectacle, React, and ReactDOM libraries, and being raring to go with a minimum of effort.
+To aid with a speedy development we've provide a simple boilerplate HTML page with a bespoke script tag that contains your entire presentation. The rest of the setup will take care of transpiling your React/ESnext code, providing Spectacle, React, and ReactDOM libraries, and being raring to go with a minimum of effort.
```

Comment:
Small grammar nit: `we've provided`

Reply:
I'd actually also remove the `a` as well before `speedy development`, just a personal preference!